cassdegrees/api/models.py

```diff
@@ -23,6 +23,18 @@ class Meta:
         unique_together = (("code", "year"),)
 
 
+# todo: Add type for elements
```

Comment:
Make sure you open an issue for this one afterwards

Reply:
Following discussion at 28/8 team meeting, list structure not to be generalised at this point. Todo will be removed.